### PR TITLE
update-external-examples to include .NET Core on RHEL7

### PR DESCRIPTION
**Pull request by tmds**
@bparees ptal

Will this make .NET Core available on Red Hat CDK?

### CODE REVIEW

**Comment by bparees** [comment]
probably not.  

in 3.11 and earlier it would have since the CDK used cluster up and cluster up included these imagestreams.

in 4.0 cluster up won't exist and i don't know how the CDK will plan to get its samples.  It may get them from the new cluster-samples-operator, so i would suggest ensuring that includes the content you want: https://github.com/openshift/cluster-samples-operator/tree/master/tmp/build

/ok-to-test
/lgtm

**Comment by bparees** [comment]
/lgtm cancel

@tmds needs hack/update-generated-bindata.sh

**Comment by tmds** [comment]
> @tmds needs hack/update-generated-bindata.sh

@bparees I had missed your comment. Should be ok now.

File: examples/image-streams/image-streams-centos7.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
           },
           {
             "annotations": {
-              "description": "Build and run .NET Core 2.0 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/redhat-developer/s2i-dotnetcore/tree/master/2.0/build/README.md.",
+              "description": "RETIRED: Build and run .NET Core 2.0 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/redhat-developer/s2i-dotnetcore/tree/master/2.0/build/README.md.",
```
[the remark this answers is not in the document]
Comment (tmds) [reply examples/image-streams/image-streams-centos7.json]: I didn't know about this. Thanks!

**Pull request merged**